tests/select/test_drop_over_night_pv.py

```diff
@@ -0,0 +1,5 @@
+from ocf_datapipes.select import DropNightPV
+
+
+def test_drop_night_pv(simple_netcdf_datapipe):
+    night_drop_pv = DropNightPV(simple_netcdf_datapipe)
```

Comment:
Good start! Would you also check the output to be what you expect it should be? As in, if any of the datapoints in the test data should be dropped, and get a batch out of the datapipe? As this currently is, I don't think it actually runs the datapipe, just constructs the datapipe graph. To run it, you would need something like 
```suggestion
def test_drop_night_pv(passiv_datapipe):
    night_drop_pv = DropNightPV(passiv_datapipe)
    batch = next(iter(night_drop_pv))
    assert <something here>
```